src/Orc.NuGetExplorer.Xaml/ViewModels/PackageDetailsViewModel.cs

```diff
@@ -66,6 +60,8 @@ internal class PackageDetailsViewModel : ViewModelBase
             UninstallPackage = new TaskCommand(OnUninstallPackageExecuteAsync, OnUninstallPackageCanExecute);
         }
 
+        private bool IsPackageApplied { get; set; }
```

Comment:
private setter?

Reply:
it's private property. I've decided to use it instead of field because of i need to Invalidate commands on this property changes. This way vm handles it automatically and correctly